src/gh.js

```diff
@@ -3,18 +3,18 @@ const github = require('@actions/github');
 const _ = require('lodash');
 const config = require('./config');
 
-const INITIAL_WAIT_TIME_BEFORE_CHECKS = 5 * 1000; // 5 seconds
+const INITIAL_WAIT_TIME_BEFORE_CHECKS = 4 * 1000; // 5 seconds
```

Comment:
nit: der kommentar dahinter ist nun out of sync... vermutlich bringts der aber auch nicht. Evlt. die const ..._MS nennen. Oder weglassen, ist ja allgemeinwissen..

Reply:
```suggestion
const INITIAL_WAIT_TIME_BEFORE_CHECKS = 4 * 1000; // 4 seconds
```